homeassistant/components/here_travel_time/sensor.py

```diff
@@ -115,6 +122,70 @@
 )
 
 
+def icon_for_travel_mode(travel_mode: str) -> str:
```

Comment:
A common pattern I see for this is to use a dict variable then get it with a default value if not found.

Reply:
Looks nicer! Thank you!